centrallix/tests/test_moneytype_04.c

```diff
@@ -0,0 +1,40 @@
+#include "obj.h"
+#include <assert.h>
+#include <stdio.h>
+
+long long
+test(char** name)
+{
+    *name = "moneytype_04 - objDataCompare";
+    MoneyType test = {70500};
+
```

Comment:
Can you add more cases for less than/greater than/equals for all these types? You can also break the tests out into multiple .c files if you feel like it's getting long.